.travis.yml

```diff
@@ -19,6 +19,8 @@ script:
 - travis-cargo build
 - travis-cargo test
 - travis-cargo doc -- --no-deps -p parking_lot -p parking_lot_core
+- travis-cargo --only nightly test -- --all-features
+- travis-cargo --only nightly doc -- --all-features --no-deps -p parking_lot -p parking_lot_core
```

Comment:
Because of the way TRAVIS_CARGO_NIGHTLY_FEATURE is set, both tests are done with `deadlock_detection` entabled.